content-src/components/DiscoveryStreamComponents/TopSites/_TopSites.scss

```diff
@@ -10,5 +10,77 @@
     .section-top-bar {
       display: none;
     }
+
+    .top-site-outer {
+      padding: 0 12px;
+    }
+
+    .top-sites-list {
+      margin: 0 -12px;
+    }
+  }
+
+  // Only show 6 cards for 2/3 and 1/3
```

Comment:
Maybe include an XXX comment here that someday the old TopSites should get an explicit prop allowing .hide-for-narrow to be passed in rather than having it render stuff and then make it display: none.